index.html

```diff
@@ -302,6 +302,20 @@ <h2 class="language">Languages</h2>
         <img src="./images/Icon-Right-Larger.svg" alt="" />
       </div>
     </section>
+<!-- ===========CONTACT-ME======= -->
+    <section class="contact-me">
+      <div class="text-field hello">
+        <h2>Contact me</h2>
+        <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
+      </div>
+      <div class="input-field">
+        <form action="https://formspree.io/f/mnqynlww" method="post">
+          <input class="input-design" type="text" name="full_name" id="full_name" placeholder="Enter your Name" maxlength="30" required>
+          <input class="input-design" type="email" name="email" id="email" placeholder="Email Address" required>
+          <textarea class="textArea" name="message"  cols="30" rows="10" minlength="" maxlength="500" placeholder="write your message here" required></textarea>
+          <input  type="submit" value="Get in touch" class="see-pro submit-btn">
+        </form>
```

Comment:
- [ ] I have noticed that the directory path of `background` does not work. I humbly suggest ensuring that the contact section has a background as shown in the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122&t=xjQQHS5zUx2k6En5-0). This will ensure that the webpage looks professional.
- [ ] Kindly also ensure that the form inputs have the `padding-left` to make the inputs resemble the ones in the Figma design 😻 
- [ ] There is a big space between the `contact-title` and `contact-text`, kindly ensure that you stick to the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122&t=xjQQHS5zUx2k6En5-0)
- [ ] In addition, I suggest increasing the `font-size` of the text so that the template design is followed.

- Your design 😎 
![backgro](https://user-images.githubusercontent.com/99394374/214562306-cc7b97fa-f823-4b74-a632-f0c3b91a639a.PNG)

- Figma design 😎 
![figm back](https://user-images.githubusercontent.com/99394374/214562484-c0e25a6d-05c3-45e6-9ae6-40c9ccbf750b.PNG)

Reply:
implemented